src/style.css

```diff
@@ -1 +1,64 @@
-/* Styles go here */
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
```

Comment:
it should always be added

* {
  box-sizing: border-box;
}

Reply:
+